applications/connecting_applications_to_services/understanding-service-binding-operator.adoc

```diff
@@ -0,0 +1,83 @@
+[id="understanding-service-binding-operator"]
+= Understanding Service Binding Operator
+include::modules/common-attributes.adoc[]
+include::modules/servicebinding-document-attributes.adoc[]
+:context: understanding-service-binding-operator
+
+toc::[]
+
+:FeatureName: Service Binding Operator
+
+Application developers need access to backing services to build and connect workloads. Connecting workloads to backing services is always a challenge as each service provider suggests a different way to access their secrets and consume them in a workload. In addition, manual configuration and maintenance of this binding together of workloads and backing services makes the process tedious, inefficient, and error-prone.
+
+{servicebinding-title} enables application developers to easily bind workloads together with Operator-managed backing services, without any manual procedures to configure the binding connection.
+
+[id="service-binding-terminology"]
+== Service Binding terminology
+This section summarizes the basic terms used in Service Binding.
+
+[horizontal]
+Service binding:: The representation of the action of providing information about a service to a workload. Examples include establishing the exchange of credentials between a Java application and a database that it requires.
+Backing service:: Any service or software that the application consumes over the network as part of its normal operation. Examples include a database, a message broker, an application with REST endpoints, an event stream, an Application Performance Monitor (APM), or a Hardware Security Module (HSM).
+Workload (application):: Any process, running within a container. Examples include a Spring Boot application, a NodeJS Express application, or a Ruby on Rails application.
+Binding data:: Information about a service that you use to configure the behavior of other resources within the cluster. Examples include credentials, connection details, volume mounts, secrets, and so on.
```

Comment:
and so on
Use only when you list a clear sequence of elements, such as "1, 2, 3, and so on" or "Monday, Tuesday, Wednesday, and so on".
When you list items that do not form a clear sequence, use appropriate descriptive wording, such as "device drivers, firmware,
and other code".
(IBM SG)